.github/workflows/test-libmaxminddb.yml

```diff
@@ -48,10 +48,16 @@ jobs:
         run: brew install libmaxminddb
         if: matrix.platform == 'macos-latest'
 
-      - name: Build with Werror and Wall
+      - name: "Work around macos arm64 homebrew directory changes"
+        if: runner.os == 'macOS' && runner.arch == 'ARM64'
+        run: |
+          echo "CFLAGS=-I/opt/homebrew/include" >> "$GITHUB_ENV"
+          echo "LDFLAGS=-L/opt/homebrew/lib" >> "$GITHUB_ENV"
+
+      - name: Build with Werror and Wall (not macOS)
```

Comment:
The `(not macOS)` can be removed

Reply:
Thanks! I force-pushed it as a single commit to reduce the commit noise.